document-parsers/langchain4j-document-parser-tesseract/src/main/java/dev/langchain4j/data/document/parser/tesseract/TesseractOcrDocumentParser.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.data.document.parser.tesseract;
+import dev.langchain4j.data.document.Document;
+import dev.langchain4j.data.document.DocumentParser;
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static dev.langchain4j.internal.Utils.*;
+
+public class TesseractOcrDocumentParser implements DocumentParser {
+    private static final String DEFAULT_LANGUAGE = "eng";
+
+    private static final String DEFAULT_DATAPATH = "src/main/resource/tessdata";
```

Comment:
This will work only in Junit, right? Not sure it is useful for prod use. I would not have a default, unless tesseract has some default path